src/appforms/tests/tests/backbone/040-view02field14field_map.js

```diff
@@ -0,0 +1,163 @@
+var assert = chai.assert;
```

Comment:
Made a start a while back (https://github.com/nialldonnellyfh/fh-forms-sdk).

Hopefully at some point we will get time to do a joint sprint effort with the sdk team to get it broken out completely. It would be cool as a standalone SDK